### PR TITLE
fix(router/atc): exports atc.schema for entities checking

### PR DESCRIPTION
**Pull request by chronolaw**
### Summary

When we did code clean for atc-router(#9331), we removed function `get_schema()` unwisely,
which is used in DB schema entity checking.

See [`schema/entities/routes.lua`](https://github.com/Kong/kong/blob/e7b6963f8c01c11232828d2709de08743708bf56/kong/db/schema/entities/routes.lua#L50).

### CODE REVIEW

**Comment by chronolaw** [comment]
> @chronolaw is it possible to add test to this? Shouldn't it e.g. runtime crash or something?

I will try to figure it out.

File: kong/router/atc.lua [hunk 1]
```diff
@@ -554,6 +554,11 @@ function _M:exec(ctx)
 end
 
 
+function _M.get_schema()
+  return CACHED_SCHEMA
+end
```
Comment (mayocream) [inline kong/router/atc.lua:559]: As it is a static table, won't be better to use the `_M.schema` format?

```suggestion
_M.schema = CACHED_SCHEMA
```

**Comment by chronolaw** [comment]
> @chronolaw is it possible to add test to this? Shouldn't it e.g. runtime crash or something?

We have unit-test in `spec/01-unit/08-router_spec.lua`, but it will not trigger schema check.

And integration test `spec/02-integration/05-proxy/02-router_spec.lua`, we only test `traditional` and `traditional_compatible`, so this is the reason.

**Comment by chronolaw** [comment]
I am trying to enable test for flavor `expressions` in `spec/02-integration/05-proxy/02-router_spec.lua`, but I get some failing cases.

Maybe we can merge it now, then add test cases in another PR.

**Comment by dndx** [comment]
If it is impossible to get the entire existing tests running under `expressions` mode, we should at least add a single test case just to catch this regression.

**Review by dndx — APPROVED** [review]

**Pull request merged**